meta-xt-domx/conf/machine/h3ulcb-4x2g-ab-xt.conf

```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
```

Comment:
Are you actually need to describe a machine? Prod devel works without this

Reply:
> 

really, if to leave the support just a machine (what is true), than, this configuration can be removed.
i'wll change it